sharding/indexHashedNodesCoordinator.go

```diff
@@ -121,6 +141,16 @@ func (ihgs *indexHashedNodesCoordinator) ComputeValidatorsGroup(
 		return nil, ErrNilRandomness
 	}
 
+	key := []byte(fmt.Sprintf(keyFormat, string(randomness), round, shardId))
+	value, ok := ihgs.consensusGroupCache.Get(key)
```

Comment:
make a separate method for this get and type assertion.

Reply:
fixed